lib/accounts/bloc/create_profile_bloc/create_profile_bloc.dart

```diff
@@ -26,12 +26,25 @@ class CreateProfileBloc extends Bloc<CreateProfileEvent, CreateProfileState> {
       var creatingProfileFunction = event.profileRole == ProfileRole.ADMIN
           ? () => accountsService.createAdminProfile(event.form)
           : () => accountsService.createBasicProfile(event.form);
-      try {
-        await creatingProfileFunction.call();
-        yield SuccessState();
-      } on Exception {
+      if (isFilled(event.form)) {
         yield ErrorState(form: event.form);
+      } else {
+        try {
+          await creatingProfileFunction.call();
+          yield SuccessState();
+        } on Exception {
+          yield ErrorState(form: event.form);
+        }
       }
     }
   }
+
+  bool isFilled(AccountProfileCreationRequest form) {
+    return form.firstName == null ||
```

Comment:
to nie zadziała jak ktoś wpisze po prostu spacje

Reply:
dodajcie do projektu jakąś paczkę z utilsami jak nie ma - np. https://pub.dev/packages/quiver i sprawdzaj po prostu czy nie jest blank (pewnie jest jakas metoda isBlank())

Reply:
dodatkowo masz metodę isFilled a tak naprawdę ona sprawdza isNotFilled